tchannel/tornado/peer.py

```diff
@@ -96,6 +100,15 @@ def __init__(self, tchannel, hostport):
         # helps avoid making multiple outgoing connections.
         self._connecting = None
 
+        # score is used to measure the performance of the peer.
+        # It will be used in the peer heap.
+        self.score = sys.maxint
+        # index records the position of the peer in the peer heap
+        self.index = -1
+
+    def __lt__(self, other):
+        return self.score < other.score
+
```

Comment:
As I mentioned yesterday, don't implement `__lt__` on something simply to compare scores conveniently. Unless the concept represented by that class has a semantic meaning of "this peer is less than that peer", they shouldn't be comparable. You're implementing the heap specifically for this use case. You can compare the score with `.score` instead of `x < y`.